content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -15,6 +15,7 @@ The following is a comprehensive list of all known [amendments](amendments.html)
 | Name                              | Introduced | Status                        |
 |:----------------------------------|:-----------|:------------------------------|
 | [OwnerPaysFee][]                  | TBD        | [In Development: TBD]( "BADGE_LIGHTGREY") |
+| [fixNFTokenRemint][]              | v1.11.0    | [In Development: TBD]("BADGE_LIGHTGREY") |
```

Comment:
I think 1.11 release is out now, should the status be something else here?

Reply:
v1.11.0 is the number from the XRPScan page. I don't know what else the number should be.

Reply:
I was looking for another label, like "voting," on the page, but all of the items on the amendments page that are currently up for a vote are labeled "In Development." I'm just following what I see on the page.

Reply:
I think this is possibly because the past amendment status are out of date and need an update too

Reply:
I think you're right. I'll look into updating all entries on the page.